duckduckgo/TrackerRadarKit
# Initial TrackerRadarKit implementation

Task/Issue URL: https://app.asana.com/0/72649045549333/1194161438970554/f
Tech Design URL: https://app.asana.com/0/481882893211075/1199405653869875/f
CC: @SlayterDev

**Description**:

This PR adds the initial implementation of TrackerRadarKit, which pulls out Tracker Radar model parsing & content blocker rule generation into its own Swift package.

Unit tests have been ported from the iOS app where applicable, and additional tests have been added to ensure good test coverage.

**Steps to test this PR**:
1. Check this branch out and ensure that unit tests run 
1. Check out the equivalent iOS PR and test for regressions



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
